Sources/SwiftDocC/Model/Rendering/Content/RenderInlineContent.swift

```diff
@@ -29,7 +29,7 @@
 ///
 /// Inline elements can be nested, for example, an inline piece of text can be wrapped in an emphasis element.
 /// Block elements cannot be nested in inline elements.
-public enum RenderInlineContent: Equatable {
+public enum RenderInlineContent: Equatable, Hashable {
```

Comment:
What's the reason for this change? I don't see anything that uses inline content in a set or dictionary.

Reply:
Without it, `[RenderInlineContent]` does not conform to `Hashable`, which prevents synthesized conformance of `RenderIndex.Node` to `Hashable`.

Reply:
Why is `RenderIndex.Node` hashable?  I wouldn't think that it's used in a set or as a dictionary key 🤔

Reply:
I just did the exact same thing 😄
I'll remove the `Hashable` conformances and switch `RenderIndex.Node` to conform to `Equatable` instead.